extensions-core/google-extensions/src/main/java/org/apache/druid/firehose/google/GoogleBlob.java

```diff
@@ -72,5 +72,10 @@ public boolean equals(Object o)
     return Objects.equals(bucket, that.bucket) &&
            Objects.equals(path, that.path);
   }
-}
 
+  @Override
+  public int hashCode()
+  {
+    return Objects.hash(getBucket(), getPath());
```

Comment:
It doesn't matter too much, but might as well do `bucket` and `path` here to match `equals`.

Reply:
Done :+1: